code/modules/mob/living/silicon/robot/robot.dm

```diff
@@ -130,7 +130,7 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
 	radio = new /obj/item/radio/borg(src)
 	common_radio = radio
 
-	init()
+	init(alien, connect_to_AI)
```

Comment:
I hate having to pass around this argument so much but I have to due to this stupid `init` proc being here.